### ISSUE TITLE
`tabs_layout="start"` is broken

### ISSUE THREAD

**Issue by bew** [nvim-neo-tree/neo-tree.nvim#848]
I'm currently on branch v2.x at https://github.com/nvim-neo-tree/neo-tree.nvim/commit/2b2f74828eeb02cf29d6b21aa32eedadadc94ca7

With the config:
```lua
require"neo-tree".setup {
  source_selector = {
    winbar = true,
    tabs_layout = "start",
  }
}
```
I get:
![Screenshot_20230407_222246](https://user-images.githubusercontent.com/9730330/230674048-5a24fae3-b76f-4dfb-b3db-223590bf5dcc.png)

The winbar is correctly set to `%{%v:lua.require'neo-tree.ui.selector'.get()%}`
and that function returns `"%#NeoTreeTabInactive#%4009@v:lua.___neotree_selector_click@%…%<%0@v:lua.___neotree_selector_click@"` (missing all tabs?)

**Comment by nhat-vo** [comment]
@bew I just pushed a fix in #852. Could you have a look to see if it fixes the issue?

**Comment by bew** [comment]
I tested your PR, works well with `tabs_layout="start"` :+1:
